src/main/java/net/dzikoysk/funnyguilds/command/user/CreateCommand.java

```diff
@@ -143,36 +145,35 @@ public void execute(PluginConfiguration config, MessageConfiguration messages, P
             return;
         }
 
+        Guild guild = new Guild(name);
+        guild.setTag(tag);
+        guild.setOwner(user);
+        guild.setLives(config.warLives);
+        guild.setBorn(System.currentTimeMillis());
+        guild.setValidity(System.currentTimeMillis() + config.validityStart);
+        guild.setAttacked(System.currentTimeMillis() - config.warWait + config.warProtection);
+        guild.setPvP(config.damageGuild);
+        guild.setHome(guildLocation);
+
         if (config.regionsEnabled) {
-            when (RegionUtils.isIn(guildLocation), messages.createIsNear);
-            when (RegionUtils.isNear(guildLocation), messages.createIsNear);
+            Region region = new Region(guild, guildLocation, config.regionSize);
 
-            if (config.createMinDistanceFromBorder != -1) {
+            if (config.createMinDistanceFromBorder > 0) {
                 WorldBorder border = player.getWorld().getWorldBorder();
-                double borderSize = border.getSize() / 2;
-                double borderX = border.getCenter().getX() + borderSize;
-                double borderZ = border.getCenter().getZ() + borderSize;
-                double distanceX = Math.abs(borderX) - Math.abs(player.getLocation().getX());
-                double distanceZ = Math.abs(borderZ) - Math.abs(player.getLocation().getZ());
+                double radius = border.getSize() / 2;
+                BoundingBox bbox = BoundingBox.of(border.getCenter().toVector(),
+                        radius - config.createMinDistanceFromBorder, 0, radius - config.createMinDistanceFromBorder);
```

Comment:
The first and second points are in 3d space. bbox should also be a box in 3d. (My subjective feeling)